torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -392,7 +393,13 @@ struct DifferentiableGraphOp {
     }
 
     detachVariables(*stack);
-    InterpreterState(f).run(*stack);
+    if (IsNewExecutorEnabled()) {
+      ExecutionPlan plan =
+          f_ptr->getPlanFor(*stack, GraphExecutor::getDefaultNumBailOuts());
+      InterpreterState(plan.code).run(*stack);
+    } else {
+      InterpreterState(legacy_f).run(*stack);
```

Comment:
unrelated to this PR: is it not expensive that we're reinstantiating `InterpreterState` on every forward ?

Reply:
I'll have to forward this to @Krovatkin

Reply:
I don't know. I'll measure it when I get a chance.